bytecode/cfg.py

```diff
@@ -74,6 +74,65 @@ def add_block(self, instructions=None):
         self._add_block(block)
         return block
 
+    def compute_stacksize(self):
+
+        def walk(block, depth, maxdepth):
```

Comment:
Can you please move walk() out of compute_stacksize()? May call it _compute_stacksize().

Does "depth" mean "stack size"? I don't know if it's worth to rename depth to size (or stack_size).